Documentation/images/demo-capabilities-gifterminal.txt

```diff
@@ -5,7 +5,8 @@
 [alban@neptune]$ # We can debug missing capabilities and lock the security      policy down to only the used capabilities.
 [alban@neptune]$ @@@@@ SKIP=1
 # Here we have a small demo app which logs failures due to     lacking capabilities. @@@@@ TERMINAL=2 DEFAULT_TERMINAL=2 PROMPT=1 PROMPT_CHAR=''
-[alban@neptune]$ cat Documentation/app-set-priority.yaml @@@@@ TERMINAL=1 DEFAULT_TERMINAL=1 PROMPT=1 PROMPT_CHAR=''
```

Comment:
Why that changes?

Reply:
Used this PR to update the GIFs

Reply:
This is anyway needed because they reference the folders which were renamed